README.md

```diff
@@ -1,2 +1,74 @@
 # k3s-local-registry
```

Comment:
this should probably be `k3d-local-registry`?

k3s is the kubernetes distro
k3d is k3s-in-docker

or, if you like analogies, kind is to kubernetes as k3d is to k3s

Reply:
whoops, yes you're right.